kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -0,0 +1,62 @@
+package com.kodilla.testing.shape;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class ShapeCollector {
+
+    private List<Shape> shapesList;
+
+    public ShapeCollector() {
+        this.shapesList = new ArrayList<Shape>();
+    }
+
+    public void addFigure(Shape shape) {
+        shapesList.add(shape);
+    }
+
+    public boolean removeFigure(Shape shape) {
+        boolean result = false;
+        if (!shapesList.isEmpty()) {
+            shapesList.remove(shape);
+            result = true;
+        }
+        return result;
+    }
+
+    public Shape getFigure(int n) {
+        return shapesList.get(n);
+    }
+
+    public List<Shape> getShapesList() {
+        return shapesList;
+    }
+
+    public void showFigures() {
+        System.out.println(shapesList);
```

Comment:
Tu ma być logika wypisywania listy, a więc w pętli for wypisujemy listę obiektów:
`StringBuilder builder = new StringBuilder();`
`for (Shape s : shapesList) {`
`builder.append(s).append(", ");`
`}`
`return builder.toString();`

Reply:
System.out.println(builder.toString());